app/javascript/src/QuickStarts/templates/template.yaml

```diff
@@ -0,0 +1,133 @@
+# Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
```

Comment:
IMO, the content PR should only include this one file, we can change the name to basicAPIintegration.yaml.
And the other files will be duplicated with [PR#2903](https://github.com/3scale/porta/pull/2903)
So this is the reason why I suggest creating content PR after PR#2903 has been merged?
cc @josemigallas here.

Reply:
Is it ok for me to rename the file without consequences elsewhere?

Let me know @josemigallas 

Cheers.

Reply:
@josemigallas I was wondering about June's question here about changing the file name?

Reply:
I think in the quick start page, we will have eight cards to show different quick start content, such as Basic API Integration, create backend, create product...... 
then here we will have eight YAML files for every content.
Yes, if you change the file name to basicAPIintegraton.yaml, then we need to update the index.js to include the new files.

Reply:
Thank you for this information. What are the standard for file naming? For example: `basicAPIintegraton.yaml` or `basicapiintegraton.yaml` or `basic-api-integraton.yaml?

Let's discuss in our stand-up.